docs/package-hub/package-hub.md

```diff
@@ -7,7 +7,11 @@ tags: [package-hub]
 
 Prophecy introduces **Package Hub,** which enables data practitioners to create and share Pipeline components.
 
-![1](img/1-ph.png)
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/699b96r7w8?seo=false?videoFoam=true" title="Getting Started With SQL Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
```

Comment:
Could you add one `<br />` to add a line of space between video and text please?